packages/terra-functional-testing/src/commands/utils/dispatchCustomEvent.js

```diff
@@ -0,0 +1,15 @@
+const dispatchCustomEvent = (name, metaData) => {
```

Comment:
i wasn't sure where to put this file. opted to put this in `commands/utils`, even though it is not attached to the terra global. open to suggestions here

Reply:
My memory is a little fuzzy on the details/use-cases of this utility function. From a fresh perspective it seems a little strange to have this independent utility function that's not used/exposed from this project. Should this be attached to a global/browser action? Here are a few of the possible approaches, I'm mainly just curious to get thoughts on which one we think makes the most sense

Custom Browser Command:

https://webdriver.io/docs/browserobject.html

```js
browser.dispatchEvent();
```

Custom Page Object:

https://webdriver.io/docs/pageobjects.html

```js
EventPage.dispatchEvent();
```

Bound to the Terra global similar to other utility functions:

```js
Terra.dispatchEvent();
```

An independent utility function:

```js
const dispatchCustomEvent = require('@cerner/terra-function-testing/lib/commands/utils/dispatchCustomEvent');

dispatchCustomEvent();
```

Reply:
I think I lean towards the `Terra.dispatchEvent` because we'll own that namespace and it is provided globally to the test run. It wouldn't require a custom import into each spec that needs it.

Reply:
good points - I agree that placing this function here without being exposed is strange. I don't see consumers calling this themselves; full stack uses this function in its after command hook to send a locale override event, to override a terra application's locale. In that sense, I'd lean towards keeping it independent.

Do we want the minimal speed benefit of being tied to open source release process? we could move this back into full stack testing, it's small. atm, the only open source use case is for terra-application to import this for testing purposes.

anyone else have thoughts?

Reply:
attaching this to the terra global; based on offline discussion
https://github.com/cerner/terra-toolkit/pull/519/commits/9d100d6f5bb139845718b9f6dcc072af1eef24c4